.gitignore

```diff
@@ -1,6 +1,7 @@
 .build/
 .idea/
 .vscode/
+minisign.*
```

Comment:
I created a key locally for testing and didn't want to check it in (not any security risk - this was just for E2E testing).